package.json

```diff
@@ -32,18 +26,54 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-simple-import-sort": "^8.0.0",
     "husky": "^8.0.1",
-    "lerna": "^5.0.0",
+    "lerna": "^5.5.2",
     "pre-commit": "^1.2.2",
     "prettier": "^2.7.1",
     "typescript": "4.7.4",
-    "vitest": "^0.22.1"
+    "vitest": "^0.23.4",
+    "@types/commander": "^2.12.2",
+    "@types/lodash.get": "^4.4.7",
+    "@types/sarif": "^2.1.4",
+    "lodash.get": "^4.4.2",
```

Comment:
This list is usually sorted you you use `add`.

Reply:
yeah merge conflicts. im just going to add an auto sort for package.json after this PR